content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -26,7 +26,8 @@ export class _StartupOverlay extends React.PureComponent {
     if (this.props.fxa_endpoint && !this.didFetch) {
       try {
         this.didFetch = true;
-        const response = await fetch(`${this.props.fxa_endpoint}/metrics-flow`);
+        const response = await fetch(`${this.props.fxa_endpoint}/metrics-flow?entrypoint=
+          activity-stream-firstrun&utm_source=activity-stream&utm_campaign=firstrun&form_type=email`);
```

Comment:
Thanks for adding these @ericawright !  👍 

@k88hudson one thing to keep in mind, if the values in https://github.com/mozilla/activity-stream/blob/27696cb3c9d88e32c03b6040102de678c99f16a5/content-src/components/StartupOverlay/StartupOverlay.jsx#L115 ever change then will need to change in this file as well.